src/adapter/src/coord/sequencer/inner.rs

```diff
@@ -237,6 +240,77 @@ impl Coordinator {
         }
     }
 
+    #[tracing::instrument(level = "debug", skip(self))]
+    pub(super) async fn sequence_create_webhook_source(
```

Comment:
What is the advantage of running this in its own function instead of `sequence_create_source`?

Reply:
The reason I split this out was `sequence_create_source` takes a `CreateSourcePlan` which requires a `SourceSinkClusterConfig`. Instead of trying to change one of these pieces to represent a `None` cluster, I figured it was easier to split the logic. I added a TODO here to merge/delete the method once we run webhook sources on `clusterd`

Reply:
SGTM